.yarnrc.yml

```diff
@@ -5,3 +5,5 @@ enableGlobalCache: false
 enableScripts: false
 
 nodeLinker: node-modules
+
+nmHoistingLimits: workspaces
```

Comment:
wdyt making it workspace-specific? https://yarnpkg.com/configuration/manifest#installConfig